### PR TITLE
Update docs post EIP-4844

### PR DESCRIPTION
**Pull request by mteam88**
Updated DA and Optimistic Rollups pages with post-4844 wording.

## Description

Updated tenses in DA article and implementation notes in optimistic rollups article.

## Related Issue

N/A, I can create one if absolutel neccessary, but I think these changes are self explanatory.

### CODE REVIEW

**Comment by mteam88** [comment]
If there are more articles that need to be updated, I'm happy to edit those as well.

File: public/content/developers/docs/data-availability/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Data availability is also a critical concern for future ["stateless"](/roadmap/s
...
+DAS is used to ensure rollup operators make their transaction data available now that [EIP-4844](/roadmap/danksharding) has been implemented. Ethereum nodes randomly sample the transaction data provided in blobs using the redundancy scheme explained above to ensure that all the data exists. The same technique could also be employed to ensure block producers are making all their data available to secure light clients. Similarly, under [proposer-builder separation](/roadmap/pbs), only the block builder would be required to process an entire block - other validators would verify using data availability sampling.
```
Comment (corwintines) [inline public/content/developers/docs/data-availability/index.md:29]:
```suggestion
DAS will to ensure rollup operators make their transaction data available after [Full Danksharding](/roadmap/danksharding/#what-is-danksharding) has been implemented. Ethereum nodes will randomly sample the transaction data provided in blobs using the redundancy scheme explained above to ensure that all the data exists. The same technique could also be employed to ensure block producers are making all their data available to secure light clients. Similarly, under [proposer-builder separation](/roadmap/pbs), only the block builder would be required to process an entire block - other validators would verify using data availability sampling.
```

This change is wrong as proto-danksharding (EIP-4844) does not introduce DAS. But, the content here was wrong in the first place, so we should revert this change, but fix what is wrong here. I suggest the above.

File: public/content/developers/docs/scaling/optimistic-rollups/index.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ lang: en
 
 Optimistic rollups are layer 2 (L2) protocols designed to extend the throughput of Ethereum's base layer. They reduce computation on the main Ethereum chain by processing transactions off-chain, offering significant improvements in processing speeds. Unlike other scaling solutions, such as [sidechains](/developers/docs/scaling/sidechains/), optimistic rollups derive security from Mainnet by publishing transaction results on-chain, or [plasma chains](/developers/docs/scaling/plasma/), which also verify transactions on Ethereum with fraud proofs, but store transaction data elsewhere.
 
-As computation is the slow, expensive part of using Ethereum, optimistic rollups can offer up to 10-100x improvements in scalability. Optimistic rollups also write transactions to Ethereum as `calldata`, reducing gas costs for users.
+As computation is the slow, expensive part of using Ethereum, optimistic rollups can offer up to 10-100x improvements in scalability. Optimistic rollups also write transactions to Ethereum as `calldata` or in [EIP-4844 blobs,](/roadmap/danksharding/) reducing gas costs for users.
```
Comment (corwintines) [inline public/content/developers/docs/scaling/optimistic-rollups/index.md:9]:
```suggestion
As computation is the slow, expensive part of using Ethereum, optimistic rollups can offer up to 10-100x improvements in scalability. Optimistic rollups also write transactions to Ethereum as `calldata` or in [blobs](/roadmap/danksharding/), reducing gas costs for users.
```

File: public/content/developers/docs/scaling/optimistic-rollups/index.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ lang: en
...
 
 ## Prerequisites {#prerequisites}
 
 You should have read and understood our pages on [Ethereum scaling](/developers/docs/scaling/) and [layer 2](/layer-2/).
 
 ## What is an optimistic rollup? {#what-is-an-optimistic-rollup}
 
-An optimistic rollup is an approach to scaling Ethereum that involves moving computation and state storage off-chain. Optimistic rollups execute transactions outside of Ethereum, but post transaction data to Mainnet as `calldata`.
+An optimistic rollup is an approach to scaling Ethereum that involves moving computation and state storage off-chain. Optimistic rollups execute transactions outside of Ethereum, but post transaction data to Mainnet as `calldata` or in [EIP-4844 blobs.](/roadmap/danksharding/)
```
Comment (corwintines) [inline public/content/developers/docs/scaling/optimistic-rollups/index.md:17]:
```suggestion
An optimistic rollup is an approach to scaling Ethereum that involves moving computation and state storage off-chain. Optimistic rollups execute transactions outside of Ethereum, but post transaction data to Mainnet as `calldata` or in [blobs](/roadmap/danksharding/).
```

File: public/content/developers/docs/scaling/optimistic-rollups/index.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Optimistic rollups rely on the main Ethereum protocol for the following:
 
 ### Data availability {#data-availability}
 
-As mentioned, optimistic rollups post transaction data to Ethereum as `calldata`. Since the rollup chain's execution is based on submitted transactions, anyone can use this information—anchored on Ethereum’s base layer—to execute the rollup’s state and verify the correctness of state transitions.
+As mentioned, optimistic rollups post transaction data to Ethereum as `calldata` or, post [EIP-4884,](/roadmap/danksharding/) in blobs. Since the rollup chain's execution is based on submitted transactions, anyone can use this information—anchored on Ethereum’s base layer—to execute the rollup’s state and verify the correctness of state transitions.
```
Comment (corwintines) [inline public/content/developers/docs/scaling/optimistic-rollups/index.md:47]:
```suggestion
As mentioned, optimistic rollups post transaction data to Ethereum as `calldata` or [blobs](/roadmap/danksharding/). Since the rollup chain's execution is based on submitted transactions, anyone can use this information—anchored on Ethereum’s base layer—to execute the rollup’s state and verify the correctness of state transitions.
```

File: public/content/developers/docs/scaling/optimistic-rollups/index.md [hunk 5]
```diff
@@ -86,16 +86,20 @@ The sequencer is different from a regular rollup operator because they have grea
...
 
 In the context of optimistic rollups, `calldata` is used to send compressed transaction data to the on-chain contract. The rollup operator adds a new batch by calling the required function in the rollup contract and passing the compressed data as function arguments. Using `calldata` reduces user fees since most costs that rollups incur come from storing data on-chain.
 
 Here is [an example](https://etherscan.io/tx/0x9102bfce17c58b5fc1c974c24b6bb7a924fb5fbd7c4cd2f675911c27422a5591) of a rollup batch submission to show how this concept works. The sequencer invoked the `appendSequencerBatch()` method and passed the compressed transaction data as inputs using `calldata`.
 
+Some rollups now use blobs to post batches of transactions to Ethereum. 
+
+Blobs are non-modifiable and non-persistent (just like `calldata`) but are not included in the blockchain's history logs. EIP-4844 blobs expire after 18 days. For more information on blobs, see [Data Availability.](/developers/docs/data-availability/)
```
Comment (corwintines) [inline public/content/developers/docs/scaling/optimistic-rollups/index.md:101]:
```suggestion
Blobs are non-modifiable and non-persistent (just like `calldata`) but are pruned from history after ~18 days. For more information on blobs, see [Danksharding](/roadmap/danksharding).
```

**Review by corwintines — COMMENTED** [review]
Just a few suggestions to clean up the content. Nice one @mteam88!

File: public/content/developers/docs/data-availability/index.md (hunk 1 above)
Comment (mteam88) [reply public/content/developers/docs/data-availability/index.md:29]: Yep, great catch. I wasn't paying attention to what this section was mentioning.

**Comment by mteam88** [comment]
Thanks @corwintines, I really appreciate the comments and suggestions.

**Review by corwintines — APPROVED** [review]
LGTM, thanks @mteam88!

File: public/content/developers/docs/data-availability/index.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ Data availability is also a critical concern for future ["stateless"](/roadmap/s
...
+DAS will to ensure rollup operators make their transaction data available after [Full Danksharding](/roadmap/danksharding/#what-is-danksharding) has been implemented. Ethereum nodes will randomly sample the transaction data provided in blobs using the redundancy scheme explained above to ensure that all the data exists. The same technique could also be employed to ensure block producers are making all their data available to secure light clients. Similarly, under [proposer-builder separation](/roadmap/pbs), only the block builder would be required to process an entire block - other validators would verify using data availability sampling.
```
Comment (wackerow) [inline public/content/developers/docs/data-availability/index.md:29]:
```suggestion
DAS will be used to ensure rollup operators make their transaction data available after [Full Danksharding](/roadmap/danksharding/#what-is-danksharding) has been implemented. Ethereum nodes will randomly sample the transaction data provided in blobs using the redundancy scheme explained above to ensure that all the data exists. The same technique could also be employed to ensure block producers are making all their data available to secure light clients. Similarly, under [proposer-builder separation](/roadmap/pbs), only the block builder would be required to process an entire block - other validators would verify using data availability sampling.
```

**Review by wackerow — APPROVED** [review]
Lgtm! Thanks @mteam88
